tests/test_matrix.py

```diff
@@ -278,3 +278,11 @@ def test_multiplyMatrix():
     # assert matPro.__str__() == ""
     """
     pass
+
+
+def test_determinant(mat='[1,2;3,4]'):
+    mat = getTokens(mat)
```

Comment:
```suggestion
    mat = getTokens('[1,2;3,4]')
```

Reply:
@8hantanu I just checked that getTokens only returns the Binary object and not any specific operator object, while I used the specific operator objects like for Multiply, Minus, etc. So, should I change this in my implementation?

Reply:
What input are you giving to `getTokens()` ?

Reply:
input was `1*4-2*3`

Reply:
Though the type is _Binary_, the `token.value` will have the correct operator symbol.